.github/workflows/ci.yml

```diff
@@ -74,8 +74,68 @@ jobs:
           command: check
           args: --manifest-path http-client/Cargo.toml --no-default-features --features tokio02
```

Comment:
Instead of checking every crate individually, could we just run `cargo check` without arguments? My experience is that that checks every crate in the workspace by default (though you can also add `--workspace` to, I assume, be sure).

Or do we need to check individually because `http-client` needs the `tokio02` feature setting?

Reply:
yeah, we did do `cargo check --workspace` before this PR but that's the problem and why I added `cargo check --manifest-path crate`, see my comment below.

Reply:
Thanks, that all makes sense now :)